src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Windows.cs

```diff
@@ -412,6 +412,33 @@ internal static unsafe Interop.Kernel32.SECURITY_ATTRIBUTES GetSecAttrs(HandleIn
             return secAttrs;
         }
 
+        internal static unsafe Interop.Kernel32.SECURITY_ATTRIBUTES GetSecAttrs(HandleInheritability inheritability, PipeSecurity pipeSecurity, out object pinningHandle)
+        {
+            pinningHandle = null;
+            Interop.Kernel32.SECURITY_ATTRIBUTES secAttrs = GetSecAttrs(inheritability);
+
+            if (pipeSecurity != null)
+            {
+                // no inherability flag was found so we need to initialize secAttrs structure.
```

Comment:
>inherability [](start = 22, length = 12)

inheritability

Reply:
also this comment should go inside the "if" shouldnt it?